### PR TITLE
Add pool page design

### LINKED ISSUE #191: Refactor ViewPool file

### LINKED ISSUE #208: Update "Pool" Page
- [ ] Pool infos tab
- [ ] Deal creation tab
- [ ] Deal funding tab
- [ ] Deal tab
- [ ] Vest tab
- [ ] Unredeemed tokens tab

Screen Shot 2022-02-02 at 11 09 38 am

### PR DESCRIPTION
**Pull request by saeta-eth**
Closes #208
Closes #191

### CODE REVIEW

File: components/Button/Button.tsx [hunk 1]
```diff
@@ -110,6 +110,25 @@ const Button = styled.button<ButtonProps>`
 				}
 			}
 		`}
+
+		${(props) =>
+		props.variant === 'quaternary' &&
+		css`
+			background: ${(props) => props.theme.colors.black};
```
Comment (Aelin-xyz) [inline components/Button/Button.tsx:117]: is this supposed to just be black and white no matter which theme you are on?

File: sections/Deals/AcceptOrReject/AcceptOrRejectBox/AcceptOrRejectBox.tsx [hunk 2]
```diff
@@ -140,104 +129,119 @@ const AcceptOrRejectDealBox: FC<AcceptOrRejectDealBoxProps> = ({
 		setInputValue(maxValue);
 	};
 
+	const handleMaxWithdrawButtonClick = () => setInputValue(userPoolBalance || 0);
+
+	const handleInputChange = (e: ChangeEvent<HTMLInputElement>) => {
```
Comment (Aelin-xyz) [inline sections/Deals/AcceptOrReject/AcceptOrRejectBox/AcceptOrRejectBox.tsx:134]: these types of changes are what I am most concerned with but as long as we are just moving around the same logic that is fine

File: sections/Deals/AcceptOrReject/AcceptOrRejectBox/AcceptOrRejectBox.tsx [hunk 3]
```diff
@@ -140,104 +129,119 @@ const AcceptOrRejectDealBox: FC<AcceptOrRejectDealBoxProps> = ({
...
+		const value = !!e.target.value.length ? parseFloat(e.target.value) : '';
+		setIsMaxValue(false);
+		setInputValue(value);
+	};
+
+	const isSecondRound = dealRedemptionData?.status === Status.OpenRedemption;
```
Comment (Aelin-xyz) [inline sections/Deals/AcceptOrReject/AcceptOrRejectBox/AcceptOrRejectBox.tsx:140]: as long as these are the only 2 stages possible here this should be ok

File: sections/Layout/PageLayout.tsx [hunk 4]
```diff
@@ -57,7 +57,11 @@ export const ContentTitle = styled.h1`
 
 export const ContentSubtitle = styled.p`
 	margin: 6px 0 0 0;
+<<<<<<< HEAD
```
Comment (Aelin-xyz) [inline sections/Layout/PageLayout.tsx:60]: this needs to be fixed obviously. the dapp can't run with these

File: sections/Layout/PageLayout.tsx [hunk 5]
```diff
@@ -57,7 +57,11 @@ export const ContentTitle = styled.h1`
...
+	color: ${(props) => props.theme.colors.textSmall};
+=======
```
Comment (Aelin-xyz) [inline sections/Layout/PageLayout.tsx:62]: same

File: components/Button/Button.tsx (hunk 1 above)
Comment (saeta-eth) [reply components/Button/Button.tsx:117]: It's a button createed by @0xlinus and it's only used on the header, so I think it's okay.

File: sections/Layout/PageLayout.tsx (hunk 5 above)
Comment (saeta-eth) [reply sections/Layout/PageLayout.tsx]: Oops, Fixed!

**Pull request merged**